neqo-crypto/src/aead_null.rs

```diff
@@ -4,8 +4,6 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-#![cfg(feature = "disable-encryption")]
-
```

Comment:
Just to double check, this can be removed, as the module is already feature flagged in `lib.rs`, correct?

https://github.com/mozilla/neqo/blob/2e4f0271fc50a8347d8c7525c66d3237a425fcfa/neqo-crypto/src/lib.rs#L11-L12

Reply:
Apparently so. (The change suggestion came from clippy.)